node_modules/col-assets/tests/util.js

```diff
@@ -413,7 +429,12 @@ var assertCreateFile = module.exports.assertCreateFile = function(client, course
   client.assets.createFile(course, title, file, opts, function(err, asset, response) {
     assert.ok(!err);
     assert.ok(asset);
-    assertAsset(asset, {'expectedCommentCount': 0, 'expectComments': true, 'expectCategories': true});
+    assertAsset(asset, {
+      'expectedCommentCount': 0,
+      'expectComments': true,
+      'expectCategories': true,
+      'expectVisible': opts.visible === false ? false : true
```

Comment:
@nicolaasmatthijs Minor but any reason not to express as `opts.visible !== false`?